module/zfs/arc.c

```diff
@@ -8035,18 +7742,12 @@ arc_init(void)
 #endif
 
 	arc_c = arc_c_min;
-	arc_p = (arc_c >> 1);
+	arc_meta = (1ULL << 32) / 4;	/* Metadata 25% of arc_c. */
+	arc_pd = (1ULL << 32) / 2;	/* MRU 50% of data. */
```

Comment:
It'd be nice to have a block comment somewhere explaining this math (I see arc_pd set to 1 << 32, and later used as: `wt * (arc_pd >> 16) >> 16;`

Reply:
Added some comments about 32-bit fractions.